.cirrus.yml

```diff
@@ -26,8 +26,8 @@ env:
     CENTOS_9_IMAGE_NAME: "centos-stream-9-v20230809"
     CENTOS_8_IMAGE_NAME: "centos-stream-8-v20230809"
     DEBIAN_IMAGE_NAME: "debian-11-bullseye-v20230809"
-    FEDORA_IMAGE_NAME: "fedora-cloud-base-gcp-38-1-6-x86-64"
-    FEDORA_PRIOR_IMAGE_NAME: "fedora-cloud-base-gcp-37-1-7-x86-64"
+    FEDORA_IMAGE_NAME: "fedora-cloud-base-gcp-39-beta-1-1-x86-64"
```

Comment:
why 39 beta? I see a released one in https://download.fedoraproject.org/pub/fedora/linux/releases/39/Cloud/x86_64/images/Fedora-Cloud-Base-GCP-39-1.5.x86_64.tar.gz  (despite the download page is broken and points to 36 as latest)

Reply:
Because Google cloud does not offer F39 GA (yet) :( see https://gcloud-compute.com/images.html .

Reply:
This has been an issue with Fedora pushing public images on GCP for a while.

For example - I reported that  [F37 was not pushed](https://pagure.io/cloud-sig/issue/402) for a while, and the download page pointed (and still does) to F36, and that ticket is still opened and seemingly unlooked at by the Cloud SIG.

Reply:
I reported the issue to `#fedora-cloud` channel, let's see if they can fix it quickly